### DISCUSSION TITLE
How would you implement drop caps with tailwindcss?

### DISCUSSION THREAD

**Discussion by zeddyblue** [tailwindlabs/tailwindcss#2351] [Help] [answered]
Does anyone know how to implement a drop cap, a big first letter at the star of a paragraph?

Similar to this https://htmldog.com/techniques/dropcaps/

**Comment by mainej** [comment, marked as answer]
Styling the first letter (along with other pseudo-elements and pseudo-classes) is supported in the [newly revised JIT mode](https://tailwindcss.com/docs/just-in-time-mode#pseudo-element-variants). Also see [this video](https://www.youtube.com/watch?v=DxcJbrs6rKk&t=265s).

**Reply by simonswiss** [reply]
Correct - you can use the `first-letter:` variant to apply styles to the first letter of the text within an element 👍

Here's a Tailwind Play example:

https://play.tailwindcss.com/bLa8dclwsJ

![image](https://user-images.githubusercontent.com/485747/122706261-dd07bb00-d29a-11eb-8c9e-62b89f99a2b5.png)
